messages.pot

```diff
@@ -0,0 +1,65 @@
+msgid ""
```

Comment:
We don't need the `messages.pot` file, that's just a temporary file to update the po files from. Once they're updated we don't have a need for it.